types/office-js-preview/index.d.ts

```diff
@@ -19919,6 +19919,455 @@ declare namespace Excel {
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        airplane = "Airplane",
+        /**
+        * Animal icon. Displays as a pawprint.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        animal = "Animal",
+        /**
+        * Apple icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        apple = "Apple",
+        /**
+        * Art icon. Displays as a paint palette.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        art = "Art",
+        /**
+        * Atom icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        atom = "Atom",
+        /**
+        * Bank icon. Displays as a building with pillars and a triangular roof.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        bank = "Bank",
+        /**
+        * Basketball icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        basketball = "Basketball",
+        /**
+        * Beaker icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        beaker = "Beaker",
+        /**
+        * Bird icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        bird = "Bird",
+        /**
+        * Book icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        book = "Book",
+        /**
+        * Bridge icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        bridge = "Bridge",
+        /**
+        * Briefcase icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        briefcase = "Briefcase",
+        /**
+        * Car icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        car = "Car",
+        /**
+        * Cat icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        cat = "Cat",
+        /**
+        * City icon. Displays as multiple tall buildings.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        city = "City",
+        /**
+        * Cloud icon.
+        *
+        * @remarks
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+        * @beta
+        */
+        cloud = "Cloud",
+        /**
+        * Constellation icon. Displays dots in the shape of the Big Dipper, with a bowl and a handle.
```

Comment:
Would using Ursa Major instead of Big Dipper work better across cultures?
```suggestion
        * Constellation icon. Displays dots in the shape of the Ursa Major.
```

Reply:
That is a great idea! I'll fix here and add to the changes spreadsheet.

Reply:
Note: "Commit suggestion" button isn't working on this PR, so I will incorporate via a new commit locally.